### PR TITLE
Welcome screen

### PR DESCRIPTION
**Pull request by sebastiancaraballo**
#### Trello board reference:

* [As a user I should be able to see a map so I can easily identify targets in the future](https://trello.com/c/CMbvOXof)

---

#### Description:

* Add https://react-leaflet.js.org/
* 

---

#### Reviewers:

* @juandmattos

---

#### Preview:

* 
Captura de pantalla 2022-12-23 a la(s) 13 33 10

### CODE REVIEW

File: src/components/Home/index.tsx [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+      <img className="smilies" src={smilies} alt="smilies" />
+      <h2>Welcome to <strong>TARGET</strong></h2>
+      <h3>Find people near you & Connect</h3>
+
+      <ul className="welcome-list">
+        <li className="item">Create a target by clicking wherever on the map, specify the ratio and a topic: Travel, Dating, Music, etc.</li>
+        <li><strong>TARGET</strong> will start a chat whenever you've a match. You can always dismiss a conversation if you're not interested.</li>
+      </ul>
+      <Button label="OK; GOT IT!" />
+    </div>
```
Comment (natokra) [inline src/components/Home/index.tsx:18]: this button has no `onClick` function. WDYT about adding some linting tool to help you with this kind of stuff? I recommend [this one](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y)

File: src/components/Home/index.tsx [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
-    Home
+   <div className="home-main">
+    <div className="home-left-container" >
+      <img className="smilies" src={smilies} alt="smilies" />
+      <h2>Welcome to <strong>TARGET</strong></h2>
+      <h3>Find people near you & Connect</h3>
+
+      <ul className="welcome-list">
+        <li className="item">Create a target by clicking wherever on the map, specify the ratio and a topic: Travel, Dating, Music, etc.</li>
+        <li><strong>TARGET</strong> will start a chat whenever you've a match. You can always dismiss a conversation if you're not interested.</li>
```
Comment (natokra) [inline src/components/Home/index.tsx:15]: WDYT about using some I18n tool like [this one](https://github.com/formatjs/formatjs)?

**Review by natokra — APPROVED** [review]
Good job! 😄 just two suggestions 👍

File: src/components/Home/index.tsx (hunk 1 above)
Comment (sebastiancaraballo) [reply src/components/Home/index.tsx:18]: Thanks, I'll create a card to work on these.

**Pull request merged**